src/js/components/Autocomplete.js

```diff
@@ -146,6 +149,12 @@ class Autocomplete extends Component {
     }
   }
 
+  handleAddShabads(e) {
+    e.preventDefault();
```

Comment:
If the type of this button is not `submit` then we don't need `e.preventDefault()`. Perhaps, it shouldn't submit button inside a form.

Reply:
Because event bubbles if dont use `e.preventDefault();` thats why.